src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -38,18 +38,20 @@ internal class ConnectedModeMigration : IConnectedModeMigration
         // Private "alias" to simplify method arguments
         private sealed class ChangedFiles : List<FilePathAndContent<string>> { }
 
-        private readonly ILogger logger;
```

Comment:
Miscellaneous: I reordered the fields declarations and argument lists to follow the convention of having the logger last.